core/src/main/scala/chisel3/IntModule.scala

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chisel3
+
+import chisel3.experimental.{BaseModule, Param}
+import chisel3.internal.BaseIntModule
+import chisel3.internal.Builder.pushCommand
+import chisel3.internal.firrtl._
+import scala.annotation.nowarn
+
+package internal {
+
+  private[chisel3] abstract class BaseIntModule(intrinsicName : String) extends BaseModule {
+    val intrinsic = intrinsicName
+  }
+
+}
```

Comment:
You can inline this into the `experimental` package. I don't think there's utility in keeping it separate given that everything inside it is private.